unix/setup.py

```diff
@@ -5,7 +5,7 @@
 
 setup(
     name="machine_stats",
-    version="2.5.4",
+    version="develop",
```

Comment:
Im curious about this as well. can you please explain to us why the change to develop?